### PR TITLE
xSY/HEEVR: clarify use of multiple algorithms

### PR DESCRIPTION
**Pull request by christoph-conrads**
**Checklist**

- [x] The documentation has been updated.
- [x] If the PR solves a specific issue, it is set to be closed on merge.

### CODE REVIEW

**Comment by christoph-conrads** [comment]
Changes to the draft in #997:
* replaced double-hyphens ("em dashes") with colons because double-hyphens look ugly in monospace fonts
* avoid a word repetition ("influences")

File: SRC/cheevr.f [hunk 1]
```diff
@@ -41,9 +41,16 @@
 *> \verbatim
 *>
 *> CHEEVR computes selected eigenvalues and, optionally, eigenvectors
-*> of a complex Hermitian matrix A.  Eigenvalues and eigenvectors can
-*> be selected by specifying either a range of values or a range of
-*> indices for the desired eigenvalues.
+*> of a real symmetric matrix A. Eigenvalues and eigenvectors can be
```
Comment (langou) [inline SRC/cheevr.f:44]: real symmetric  => complex Hermitian

File: SRC/zheevr.f [hunk 2]
```diff
@@ -41,9 +41,16 @@
 *> \verbatim
 *>
 *> ZHEEVR computes selected eigenvalues and, optionally, eigenvectors
-*> of a complex Hermitian matrix A.  Eigenvalues and eigenvectors can
-*> be selected by specifying either a range of values or a range of
-*> indices for the desired eigenvalues.
+*> of a real symmetric matrix A. Eigenvalues and eigenvectors can be
```
Comment (langou) [inline SRC/zheevr.f:44]: real symmetric  => complex Hermitian

**Review by langou — CHANGES_REQUESTED** [review]

File: SRC/cheevr.f (hunk 1 above)
Comment (christoph-conrads) [reply SRC/cheevr.f]: fixed. Thanks for proof-reading.

**Review by langou — APPROVED** [review]

**Pull request merged**